src/main/java/com/mraof/minestuck/network/AnthvilPacket.java

```diff
@@ -0,0 +1,41 @@
+package com.mraof.minestuck.network;
+
+import com.mraof.minestuck.blockentity.machine.AnthvilBlockEntity;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.server.level.ServerPlayer;
+
+public class AnthvilPacket implements PlayToServerPacket
+{
+	private final BlockPos pos;
```

Comment:
Instead of sending a position through the packet (to a block that the player may or may not reach), I suggest instead making use of `ContainerLevelAccess` from `machineContainerMenu.getPosition()` (See `SendificatorPacket` for an example)